Detectors/TRD/workflow/src/TRDDigitizerSpec.cxx

```diff
@@ -99,8 +99,11 @@ class TRDDPLDigitizerTask : public o2::base::BaseDPLDigitizer
         LOG(INFO) << "For collision " << collID << " eventID " << part.entryID << " found TRD " << hits.size() << " hits ";
 
         mDigitizer.process(hits, digits, labels);
+        mDigitizer.flush(digits, labels);
+        LOG(debug) << "For collision " << collID << " eventID " << part.entryID << " found TRD " << hits.size() << " hits with digits of size :" << digits.size();
```

Comment:
I would also remove all of this printing, we now what the problem is and how to fix it.

Reply:
removed